src/main/java/com/seller/panel/model/Companies.java

```diff
@@ -11,11 +12,13 @@
 public class Companies {
 
     @Id
-    @GeneratedValue(strategy = GenerationType.AUTO)
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
+    @NotBlank(message = "Provide company name")
     private String name;
 
+    @NotBlank(message = "Provide company code")
```

Comment:
From where we would get code?

Reply:
We need to ask at the time of registration

Reply:
We cannot ask from user, it would unneccessary logic and handling to make user understand what it is.

Reply:
okay, let it be in companies table, I removed from the request and this validation